server/src/lib/data.ts

```diff
@@ -0,0 +1,95 @@
+import {db} from "./db"
+import BlogSchema from "../Schema/Blog"
+import {z} from "zod"
+import { PrismaClient, Category } from '@prisma/client';
+
+
+// getting all the available blogs in the databse 
+export async function getBlog(){
+    return db.blog.findMany()
+}
+
+export async function getBlogByAuthorId(authorId: string) {
```

Comment:
1. this function returns multiple blogs, change the name to `getBlogsByAuthorId(authorId:string)` to prevent misunderstanding
2. This returns nothing, it should be changed to
`db.blog.create({
data:{
author:{
connect:{
id:authorId
}
}
}
})`

this is relational query in prisma